admin/src/templates/index.html

```diff
@@ -12,9 +12,8 @@
       <div class="body container">
         <div class="navigation">
           <ul>
-            <li ng-class="{'active-nav': checkActive('settings')}" mm-auth="can_configure"><a ui-sref="settings.basic" translate>Settings</a></li>
+            <li ng-class="{'active-nav': checkActive('display')}" mm-auth="can_configure"><a ui-sref="display.date-time" translate>admin.display</a></li>
```

Comment:
You've replaced the old settings link which means there's now no way to set the default locales in the app. Add back the Settings link as the first option in this list. You'll have to modify the settings.html and routes and so on because now it's just a single page rather than a page with tabs.